src/main/scala/org/broadinstitute/dsde/firecloud/service/TrialService.scala

```diff
@@ -42,74 +43,60 @@ final class TrialService
   extends Actor with LazyLogging with SprayJsonSupport {
 
   override def receive = {
-    case EnableUsers(userInfo, users) => enableUsers(userInfo, users) pipeTo sender
-    case DisableUsers(userInfo, users) => disableUsers(userInfo, users) pipeTo sender
-    case EnrollUser(userInfo) => enrollUser(userInfo) pipeTo sender
-    case TerminateUsers(userInfo, users) => terminateUsers(userInfo, users) pipeTo sender
+    case EnableUsers(managerInfo, userEmails) => enableUsers(managerInfo, userEmails) pipeTo sender
+    case DisableUsers(managerInfo, userEmails) => disableUsers(managerInfo, userEmails) pipeTo sender
+    case EnrollUser(managerInfo) => enrollUser(managerInfo) pipeTo sender
+    case TerminateUsers(managerInfo, userEmails) => terminateUsers(managerInfo, userEmails) pipeTo sender
```

Comment:
This is where to enforce that only campaign managers can execute these actions. This is a must-do before merging this PR, and sorry for missing it in previous review. Mix in `PermissionsSupport`, and then wrap the methods in `asTrialCampaignManager`:

```
case EnableUsers(managerInfo, userEmails) => asTrialCampaignManager{enableUsers(managerInfo, userEmails)} pipeTo sender
case DisableUsers(managerInfo, userEmails) => asTrialCampaignManager{disableUsers(managerInfo, userEmails)} pipeTo sender
case TerminateUsers(managerInfo, userEmails) => asTrialCampaignManager{terminateUsers(managerInfo, userEmails)} pipeTo sender
```
search the codebase for `asAdmin` to see a parallel usage. And note that `enrollUser` should NOT be wrapped in `asTrialCampaignManager`.